src/libraries/Microsoft.PowerFx.Core/Public/Types/OptionSetValueType.cs

```diff
@@ -72,9 +72,11 @@ internal override void DefaultExpressionValue(StringBuilder sb)
 
             if (info != null && info.DisplayNameProvider.LogicalToDisplayPairs.Any())
             {
+                var firstOrderedValue = info.DisplayNameProvider.LogicalToDisplayPairs.OrderBy(x => x.Key.Value).First();
```

Comment:
>var firstOrderedValue = info.DisplayNameProvider.LogicalToDisplayPairs [](http://example.com/codeflow?start=16&length=70)

Suggestion to avoid getting info.DisplayNameProvider.LogicalToDisplayPairs twice: move this to before the if:

    var firstOrdered = info
        ?.DisplayNameProvider
        .LogicalToDisplayPairs
        .OrderBy(x => x.Key.Value)
        .FirstOrDefault();

Then check if firstOrdered is not null (follow the if true branch) or not (follow the else branch)